README.md

```diff
@@ -151,6 +151,33 @@ Acme\ContactBundle\Entity\Contact:
         strategy: AUTO
 ```
 
+Annotations Version:
+``` php
+<?php
+// src/Acme/ContactBundle/Entity/Contact.php
+
+namespace Acme\ContactBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Mremi\ContactBundle\Entity\Contact as BaseContact;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="contact")
+ */
+class Contact extends BaseContact
+{
+    /**
+     * @var integer
```

Comment:
Should be `int`, can you also fix it above please? Please squash your commits (or amend the existing one) to have only one. Thx